src/osvr/Macros.cmake

```diff
@@ -34,10 +39,10 @@ macro(osvr_add_library)
 
     set_target_properties(${LIBNAME_FULL} PROPERTIES
         FOLDER "${LIB_FOLDER}")
-
-    generate_export_header(${LIBNAME_FULL}
-        BASE_NAME ${EXPORT_BASENAME}
-        EXPORT_FILE_NAME Export.h)
+                            
```

Comment:
hehe, nope, I tend to use notepad++ for CMake scripts and it doesn't trim trailing whitespace automatically (there is a macro though). I removed trailing spaces